ddtrace/contrib/mysqldb/patch.py

```diff
@@ -109,3 +114,10 @@ def patch_conn(conn, *args, **kwargs):
     wrapped = TracedConnection(conn, pin=pin, cfg=config.mysqldb)
     pin.onto(wrapped)
     return wrapped
+
+
+def _convert_tags(attr):
```

Comment:
This is a great candidate for deduplication with a utility function in `contrib/trace_utils.py`.